README.md

```diff
@@ -65,6 +64,18 @@ You can generate the docstring HTML locally by running the following in the root
 
 ```docfx docs/docfx.json --serve```
 
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues/new).
```

Comment:
Missing the entry for the QLDB Forum that's in the Python version.

Reply:
Added